app/controllers/journeys/income/AnyOtherIncomeController.scala

```diff
@@ -30,41 +30,42 @@ import views.html.journeys.income.AnyOtherIncomeView
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class AnyOtherIncomeController @Inject()(override val messagesApi: MessagesApi,
-                                         sessionRepository: SessionRepository,
-                                         navigator: Navigator,
-                                         identify: IdentifierAction,
-                                         getData: DataRetrievalAction,
-                                         formProvider: AnyOtherIncomeFormProvider,
-                                         val controllerComponents: MessagesControllerComponents,
-                                         view: AnyOtherIncomeView)
-                                        (implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+class AnyOtherIncomeController @Inject() (override val messagesApi: MessagesApi,
+                                          sessionRepository: SessionRepository,
+                                          navigator: Navigator,
+                                          identify: IdentifierAction,
+                                          getData: DataRetrievalAction,
+                                          formProvider: AnyOtherIncomeFormProvider,
+                                          val controllerComponents: MessagesControllerComponents,
+                                          view: AnyOtherIncomeView)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with I18nSupport {
 
-  val form = formProvider()
+  def isAgentString(isAgent: Boolean) = if (isAgent) "agent" else "individual"
 
-  def onPageLoad(taxYear: Int, mode: Mode): Action[AnyContent] = (identify andThen getData) { //TODO add requireData SASS-5841
+  def onPageLoad(taxYear: Int, mode: Mode): Action[AnyContent] = (identify andThen getData) { // TODO add requireData SASS-5841
     implicit request =>
-
+      val isAgent = isAgentString(request.user.isAgent)
```

Comment:
I would probably suggest renaming the method `isAgentString` to something that represents what it is doing, i.e. getting the type of auth user. The `is` in the name suggests it returns a boolean, and `string` suffixed on the end signifies it is a string (which we usually don't need to do with IDEs these days).

I would recommend (for readability purposes) renaming `isAgentString` to `authUserType` (or wtte), making it private and sticking it at the bottom of the class. You then don't need to assign it to a variable (like you have done on line 48) as it is then self-explanatory, and you can just inline it like you have e.g. on line 57.

Your FormProvider classes can then accept a `authUserType: String` instead of `isAgentString: String`

Reply:
This is a good point, I will make this change, thanks